### PR TITLE
Port 7095 create gcp terraform

### PR DESCRIPTION
**Pull request by matan84**

### CODE REVIEW

File: modules/gcp_helpers/authorization/main.tf [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+resource "google_organization_iam_custom_role" "ocean_integration_iam_org_role" {
+  title       = "The Ocean Integration organization role"
+  role_id     = var.role_id
+  permissions = var.permissions
+  org_id      = var.organization
+}
+
+resource "google_service_account_iam_binding" "ocean_integration_iam_org_binding" {
+  service_account_id = google_service_account.ocean_integration_service_account.name
```
Comment (stavbernazport) [inline modules/gcp_helpers/authorization/main.tf:15]: I don't know it it's intentional but just making sure it's correct 
service_acount_**id** = service_account.**name**

File: examples/gcp_integration_with_real_time/variables.tf [hunk 2]
```diff
@@ -0,0 +1,49 @@
+variable "port_client_id" {
+  type        = string
+  description = "The Port client id"
```
Comment (stavbernazport) [inline examples/gcp_integration_with_real_time/variables.tf:3]: Add sensitive

File: examples/gcp_integration_with_real_time/variables.tf [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+}
+variable "port_client_secret" {
+  type        = string
```
Comment (stavbernazport) [inline examples/gcp_integration_with_real_time/variables.tf:6]: Add sensitive

**Review by stavbernazport — APPROVED** [review]
LGTM besides minor comments

File: modules/gcp_helpers/authorization/main.tf (hunk 1 above)
Comment (matan84) [reply modules/gcp_helpers/authorization/main.tf:15]: image
Yup, seems like bad naming on their side

**Pull request merged**